### ISSUE TITLE
java.lang.NoSuchMethodError

### ISSUE THREAD

**Issue by mfikes** [replete-repl/replete-android#13]
Caught by Google's pre-release fuzz testing on a P8 Lite running Android 5.0. Maybe Android 5.0 should be excluded? Dunno.

```
01-19 13:17:16.245: E/AndroidRuntime(27272): java.lang.NoSuchMethodError: No virtual method startActionMode(Landroid/view/ActionMode$Callback;I)Landroid/view/ActionMode; in class Landroid/view/View; or its super classes (declaration of 'android.view.View' appears in /system/framework/framework.jar:classes2.dex)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at replete.MainActivity$onCreate$2.onItemClick(Main.kt:393)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at android.widget.AdapterView.performItemClick(AdapterView.java:303)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at android.widget.AbsListView.performItemClick(AbsListView.java:1151)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at android.widget.AbsListView$PerformClick.run(AbsListView.java:3065)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at android.widget.AbsListView$3.run(AbsListView.java:3878)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at android.os.Handler.handleCallback(Handler.java:739)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at android.os.Handler.dispatchMessage(Handler.java:95)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at androidx.test.espresso.base.Interrogator.a(Interrogator.java:19)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at androidx.test.espresso.base.UiControllerImpl.a(UiControllerImpl.java:164)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at androidx.test.espresso.base.UiControllerImpl.a(UiControllerImpl.java:156)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at androidx.test.espresso.base.UiControllerImpl.a(UiControllerImpl.java:136)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at androidx.test.espresso.action.Tap$1.b(Tap.java:6)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at androidx.test.espresso.action.GeneralClickAction.perform(GeneralClickAction.java:22)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at androidx.test.espresso.ViewInteraction$SingleExecutionViewAction.perform(ViewInteraction.java:9)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at androidx.test.espresso.ViewInteraction.a(ViewInteraction.java:79)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at androidx.test.espresso.ViewInteraction.a(ViewInteraction.java:96)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at androidx.test.espresso.ViewInteraction$1.call(ViewInteraction.java:3)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at android.os.Handler.handleCallback(Handler.java:739)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at android.os.Handler.dispatchMessage(Handler.java:95)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at android.os.Looper.loop(Looper.java:135)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at android.app.ActivityThread.main(ActivityThread.java:5538)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at java.lang.reflect.Method.invoke(Native Method)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at java.lang.reflect.Method.invoke(Method.java:372)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
01-19 13:17:16.245: E/AndroidRuntime(27272): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
```

**Comment by roman01la** [comment]
Looks like it's calling API 23 method which is not present on 5.0, I'll fix that

**Comment by roman01la** [comment]
Resolved in 98ac737c, confirmed to work on 5.1
<img width="341" alt="screen shot 2019-01-20 at 2 50 15 am" src="https://user-images.githubusercontent.com/1355501/51433913-87a8c900-1c5e-11e9-91e6-8998f8ec528c.png">
